skydive-grafana/README.md

```diff
@@ -0,0 +1,59 @@
+## Skydive for OCP and Grafana with skydive DataSource plugin
+
+This repository is based in the Skydive Project (https://github.com/skydive-project/skydive), an open source real-time network topology and protocols analyzer.
+
+Use the following instructions to deploy the full stack, Skydive + Grafana with Skydive plugin. Note that the Skydive Agent is configured to get information from the SDN using ovs-multitenant plugin for this particular example:
+
+      # oc new-project skydive
+      # oc create -f https://raw.githubusercontent.com/redhat-cop/containers-quickstarts/skydive-grafana/master/kube/skydive-template.json
```

Comment:
There's also a bug here (or maybe something hasn't been merged yet).  The template or even sub-folder doesn't exist.